spk/transmission/src/service-setup.sh

```diff
@@ -5,7 +5,7 @@ PATH="${SYNOPKG_PKGDEST}/bin:${PYTHON_DIR}/bin:${PATH}"
 CFG_FILE="${SYNOPKG_PKGVAR}/settings.json"
 TRANSMISSION="${SYNOPKG_PKGDEST}/bin/transmission-daemon"
 
-GROUP="sc-download"
+GROUP="sc-transmission"
```

Comment:
You can remove this line the default is `sc-{packagename}`

Reply:
Actually can you leave it as `sc-download` for backwards compatibility with DSM 6?